### ISSUE TITLE
unable to update to 2.7.0

### ISSUE THREAD

**Issue by kaunix** [mrlt8/docker-wyze-bridge#1101]
### Discussed in https://github.com/mrlt8/docker-wyze-bridge/discussions/1100

<div type='discussions-op-text'>

<sup>Originally posted by **kaunix** January 19, 2024</sup>
ran docker-compose stop, rm, pull, up -d, docker image prune. relogged in and authed. and it still says 2.6.0. did it multiple times and got the same thing...

any ideas?
</div>

**Comment by joevoisin** [comment]
looking at the logs, it does look like it is 2.7.0.  likely forgot to update the 2.6 to 2.7 in part of the code..

joe@nuc3:/home/docker/wyze$ docker-compose logs
Attaching to wyze-bridge
wyze-bridge    | 🚀 DOCKER-WYZE-BRIDGE v2.6.0 X86_64 2.7.0 BUILD [2024-01-11t04:20:13.342z]

**Comment by talormanda** [comment]
Yes same issue here. The code still mentions 2.6 when you are really on 2.7. Build still shows 2.7.0 for me but the bottom of the webpage references that I'm on 2.6.

![image](https://github.com/mrlt8/docker-wyze-bridge/assets/10092964/0796e062-c75f-4a3b-8493-eebfa38fabd4)

**Comment by jasondefuria** [comment]
Same issue here!

**Comment by jshg46** [comment]
same here, clicked update and it claims 270 is available.

**Comment by jjyu420** [comment]
Know bug in 2.70 , you are Likely on 2.7 already.
